src/main/java/uk/gov/pay/directdebit/payments/api/PaymentViewResponse.java

```diff
@@ -24,19 +24,19 @@
     @JsonProperty("page")
     private Long page;
     @JsonProperty("results")
-    private List<PaymentResponse> paymentViewResponses;
+    private List<CollectPaymentResponse> paymentViewResponses;
     @JsonProperty("_links")
     private ViewPaginationBuilder paginationBuilder;
 
-    public PaymentViewResponse(String gatewayExternalId, Long total, Long page, List<PaymentResponse> paymentViewResponses) {
+    public PaymentViewResponse(String gatewayExternalId, Long total, Long page, List<CollectPaymentResponse> paymentViewResponses) {
```

Comment:
I would choose `PaymentResponse` over `CollectPaymentResponse` as 1) `CollectPaymentResponse` doesn't quite make English sense as a response for viewing a payment 2) `PaymentResponse` still makes sense as a response to collecting a payment.

Reply:
[Already spoke to Oz but just for posterity]

Me and @danworth agreed this would be renamed in a separate commit.